src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Core/Process/Process.ts

```diff
@@ -26,17 +26,17 @@ export interface IStateData {
 }
 
 export interface IProcessProperties {
-    hasCommentColumn : boolean;
+    hasCommentColumn? : boolean;
```

Comment:
I understand that it's preferable to have all these booleans behave analogously and default to `false`. But why did you call this "improve type"?